sandpack-client/src/clients/node/index.ts

```diff
@@ -1,5 +1,7 @@
 /* eslint-disable no-console,@typescript-eslint/no-explicit-any,prefer-rest-params,@typescript-eslint/explicit-module-boundary-types */
 
+import { Buffer } from "buffer";
```

Comment:
@danilowoz i'm not super familiar with the code base, could you approve that `buffer` is only one dependency that was polyfill with `@esbuild-plugins/node-globals-polyfill`? i do not see other things

Reply:
I believe so